core/domain/exp_domain.py

```diff
@@ -89,6 +91,12 @@
 # Represents answers which led to the 'default outcome' of an interaction,
 # rather than belonging to a specific answer group.
 DEFAULT_OUTCOME_CLASSIFICATION = 'default_outcome'
+# The fields in ExplorationChange object.
+OPTIONAL_ATTRIBUTE_NAMES = [
```

Comment:
Either group this with the CMD_* stuff above, or make this a class-level constant in ExplorationChange. Also consider renaming to OPTIONAL_CMD_ATTRIBUTE_NAMES -- otherwise the name is very generic.

Reply:
Done.